app/components/Buttons.tsx

```diff
@@ -0,0 +1,17 @@
+export default function PrimaryButton() {
+  return (
+    <button className="bg-gray-800 rounded-3xl py-1 px-8 m-1 text-white hover:bg-emerald-50 hover:text-gray-800 uppercase font-semibold">Join</button>
```

Comment:
You probably want to pass in `children` so that the user of the button can pass in whatever label/content they want:

```suggestion
    <button className="bg-gray-800 rounded-3xl py-1 px-8 m-1 text-white hover:bg-emerald-50 hover:text-gray-800 uppercase font-semibold">{children}</button>
```

Reply:
This is done per suggestion!